auth/basics/android.md

```diff
@@ -46,19 +53,45 @@ skygear.signupWithEmail(email, password, new AuthResponseHandler() {
     }
 
     @Override
-    public void onAuthFail(String reason) {
-        Log.i("Skygear Signup", "onAuthFail: Fail with reason: " + reason);
+    public void onAuthFail(Error error) {
+        Log.i("Skygear Signup", "onAuthFail: Fail with reason: " + error.getMessage());
     }
 });
 ```
 
-## Logging in
+### Anonymous user
 
-The following shows how to log in a user with username and password.
+Skygear support users to sign up anonymously. However, they can only read data from the public database, but cannot perform operations such as saving data.
+
+To create an anonymous user, call `skygear.signupAnonymously()` as shown below:
 
 ```java
-Container skygear = Container.defaultContainer(this);
+skygear.signupAninymously(new AuthResponseHandler() {
+    @Override
+    public void onAuthSuccess(User user) {
+        Log.i("Skygear Signup", "onAuthSuccess: Got token: " + user.accessToken);
```

Comment:
I believe it will be better to explain what an `access token` is, perhaps somewhere above as an "Overview" of authentication